docs/dev/reference/hooks/prepareFormData.md

```diff
@@ -35,7 +35,42 @@ distribution or data storage.
 
 
 ## Example
+{{< tabs groupId="attribute-annotation-yaml-php" >}}
```

Comment:
```suggestion
{{< tabs groupId="prepareFormData" >}}
```

This is an existing group ID intended for tabs with "Attribute, Annotation, YAML, PHP". We need a unique group ID for this one.